ironfish-cli/src/commands/evm/balance.ts

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { isValidAddress } from '@ethereumjs/util'
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class EvmBalanceCommand extends IronfishCommand {
+  static description = `Displays an account's unshielded balance`
+
+  static flags = {
+    ...RemoteFlags,
+    address: Flags.string({
+      char: 'a',
+      description: 'EVM address of account to get unshielded balance for',
+      required: true,
+    }),
+    confirmations: Flags.integer({
+      required: false,
+      description: 'Minimum number of blocks confirmations for a transaction',
+    }),
+    // TODO(hughy): add support for custom tokens
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(EvmBalanceCommand)
+
```

Comment:
Could use default account if not provided

Reply:
I wasn't sure whether to use accounts or addresses for this command. I went with addresses so that we'd be able to use it to check balances for accounts that aren't in our wallets